### ISSUE TITLE
Is it possible to have openslide-xarray?

### ISSUE THREAD

**Issue by Mr-Milk** [swamidasslab/tiffslide-xarray#2]
Thanks for developing this package! I wonder if it is possible to use Openslide to create a lazy-loaded datatree?

**Comment by swamidass** [comment]
Yes that is possible, but it won't be quite as performant. Are you willing and able to help?

**Comment by Mr-Milk** [comment]
I would love to help, but I'm not familiar with xarray. I can have a try.

**Comment by swamidass** [comment]
Alright. I'm busy this next month, but maybe october we can pick this up again?

**Comment by Mr-Milk** [comment]
Sounds good to me!
